### PR TITLE
Updated docs to use the term zoo keeper correctly

### LINKED ISSUE #132: Update docs to use the term "ZooKeeper" correctly
# Description

"ZooKeeper" should always be written that way, we still have instances of "zookeeper" in the docus.

Occurrences of other usages should be replaced with the correct one. 

## Todo:

- [ ]  In the file `.github/vale/styles/Klaw/branding.yml`, add this Vale style:  `"(?i)zookeeper": ZooKeeper` 
    -  this will find the wrong spelling of ZooKeeper and mark them as error
- [ ]  run `npm run spell:error`

This will give you a report of errors and help you find occurrences of the wrong use in the text.

The log will show you the file where the word was found as well as the line number, for example:

Screenshot 2023-10-05 at 09 34 48

- [ ]  update the usage to correct branding in text in markdown files

The `npm run spell:error` should be green before making a PR.

### PR DESCRIPTION
**Pull request by csengineer1990**
Update docs to use the term zoo keeper correctly. I have made the required changes according to the given issue.

Notes for Reviewers
Please check the above changes if I did anything wrong please let me know.

This PR fixes #132

### CODE REVIEW

**Review by programmiri — COMMENTED** [review]
Thanks for your PR! 

I left a comment. 

Also, the pipeline is failing for an error in the file `vale.md`. For this, could you remove the whole sentence where the error is? (`Adding "ZooKeeper" for example will still show an error for "zookeeper", which is something we want.`). This was an example that we don't need anymore now that "ZooKeeper" is part of the branding rules.

File: package-lock.json
[the remark this answers is not in the document]
Comment (csengineer1990) [reply package-lock.json:1]: ok, mam. I have made the changes. could you please check and if I did anything wrong please let me know. Thank you.

[the remark this answers is not in the document]
Comment (csengineer1990) [reply package-lock.json:1]: "Hi Mam, I've rechecked, and there aren't any changes in the 'package-lock.json' file on this branch. It seems to be in the correct state as expected.

[the remark this answers is not in the document]
Comment (programmiri) [reply package-lock.json:1]: Hi! If you check the diff here in github: https://github.com/Aiven-Open/klaw-docs/pull/137/files it shows changes. 

Could you check again? Let me know if you can't find anything, in that case I could (if it's ok for you!) commit to your branch and remove that change.

[the remark this answers is not in the document]
Comment (csengineer1990) [reply package-lock.json:1]: ok mam I will check again and correct it.

**Comment by csengineer1990** [comment]
Hi mam, I have removed unnecessary changes and made the required changes in the package.json file. Please check and tell me if is there any issue.

**Comment by programmiri** [comment]
Thank you @csengineer1990! The only thing left to do is to update your branch, it has a conflict with `main`.

**Review by programmiri — APPROVED** [review]
Looks good, going to merge now :) We appreciate your contribution 🙏

**Comment by csengineer1990** [comment]
> Looks good, going to merge now :) We appreciate your contribution 🙏

Thank you so much! I'm glad to hear it looks good. Your appreciation means a lot 🙏.

**Pull request merged**